components/edit-collective/sections/Policies.js

```diff
@@ -14,17 +14,14 @@ import { parseToBoolean } from '../../../lib/utils';
 import Container from '../../Container';
 import { Flex } from '../../Grid';
 import MessageBoxGraphqlError from '../../MessageBoxGraphqlError';
+import RichTextEditor from '../../RichTextEditor';
 import StyledButton from '../../StyledButton';
 import StyledInputField from '../../StyledInputField';
 import StyledSelect from '../../StyledSelect';
-import StyledTextarea from '../../StyledTextarea';
 import { H3, P } from '../../Text';
 
 import { getSettingsQuery } from './EditCollectivePage';
 
-const EXPENSE_POLICY_MAX_LENGTH = 16000; // max in database is ~15,500
-const CONTRIBUTION_POLICY_MAX_LENGTH = 3000; // 600 words * 5 characters average length word
```

Comment:
This is something that was recently introduced and I think we want to keep it.

Reply:
Sure, I will add `showCount` prop to RichTextEditor 👍🏻 .

Reply:
fixed